data_processors/pipeline/lambdas/star_alignment.py

```diff
@@ -0,0 +1,151 @@
+import json
+
+try:
+    import unzip_requirements
+except ImportError:
+    pass
+
+import os
+import django
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'data_portal.settings.base')
+django.setup()
+
+# ---
+import logging
+from libumccr import libjson, libdt, aws
+from libumccr.aws import libssm
+from libumccr.aws.liblambda import LambdaInvocationType
+
+from data_portal.models import Workflow
+from data_processors.pipeline.domain.config import STAR_ALIGNMENT_LAMBDA_ARN
+from data_processors.pipeline.domain.workflow import ExternalWorkflowHelper, WorkflowType
+from data_processors.pipeline.services import workflow_srv, libraryrun_srv
+
+# TODO: need to find sensible data for those. Could be blank to start with and updated once we receive workflow events?
+WFL_ID = "N/A"
+WFV_ID = "N/A"
+WF_VERSION = "N/A"
```

Comment:
With, now, new refactored Workflow constraint made, they become nullable. So, perhaps, leave these 3 out all together..? I will follow up on workflow update side, during even handling.

Reply:
What about the workflow name? 
Did we decide to leave it to the "client" or control it from the orchestration?
(naming conventions are not needed here, but may be easier to enforce from the portal side)

Reply:
For first cut, let us leave it to client, Flo. i.e NF stack job submission Lambda i.e. "facade". After the first spin, we shall regroup and discuss. So, we will go with the convention use there - which is not far too off, IMO...

I am kind of thinking - making direct interface to Batch job submission from Portal, instead of "facade" - which gives us/Portal more control (and, scwatt doesn't need to field any change requests to "facade"). Potentially reduce, sort of, double handling...